app/views/challenges/_header.html.erb

```diff
@@ -1,7 +1,7 @@
 <div id='challenge-header' class='<%= controller_name %>'>
   <div class="container">
     <div class="row">
-      <div class="col-sm-10 col-sm-offset-1">
+      <div class="col-md-12 col-sm-12 padding-horizontal-none">
```

Comment:
Sorry about that -- you're totallly right. I've removed all but the container (which gives us the correct margins) and added `.padding-horizontal-none` so that the bar is full-width on xs screens.